src/dcd_mapping/vrs_map.py

```diff
@@ -374,40 +408,78 @@ def _map_genomic(
     :param align_result: The transcript selection information for a score set
     :return: VRS mapping object if mapping succeeds
     """
-    pre_mapped_hgvs_strings = _create_pre_mapped_hgvs_strings(
-        row.hgvs_nt,
-        AnnotationLayer.GENOMIC,
-        alignment=align_result,
-    )
-    post_mapped_hgvs_strings = _create_post_mapped_hgvs_strings(
-        row.hgvs_nt,
-        AnnotationLayer.GENOMIC,
-        alignment=align_result,
-    )
+    if (
+        row.hgvs_nt in {"_wt", "_sy", "="}
+        or "fs"
+        in row.hgvs_nt  # TODO I think this line can be taken out, I don't think fs nomenclature can be used for nt anyway
```

Comment:
Were there any score sets you encountered where this was a problem? I am not well positioned to answer, but figured I'd comment in case we should discuss with someone.